RedfishEventListener_v1.py

```diff
@@ -207,10 +207,7 @@ def parse_list(string: str):
         config['keyfile'] = parsed_config.get('CertificateDetails', 'keyfile')
 
     # Subscription Details
-    if parsed_config.has_section("SubsciptionDetails") and parsed_config.has_section("SubscriptionDetails"):
```

Comment:
This section needs to remain in place for backwards compatibility. We had a spelling error in the config file for the original tool and wish to not break those users.

Reply:
@jautor any opinion on this? It's not unprecedented that we forced config file changes with a tool update (we changed a number of options to be proper arrays rather than a comma-delimited list). At the time when we noticed spelling error, we figured it would be easy enough to support both forms.

Reply:
Had a chat with @jautor about this. We think we should keep this code in place, but embellish this code block with comments about the history of the spelling error in the config file and the goal to keep for backwards compatibility.